storybookjs/storybook
# Fix fullscreen mobile UI

Issue: #17808 

## What I did
- I removed the desktop-only component surrounding the fullscreen button on mobile UI. 
- I hid the mobile nav UI when fullscreen is active.

## How to test
Open browser and change size to width < 600px. Then check if the fullscreen button is still there and whether it works.


If your answer is yes to any of these, please make sure to include it in your PR.
